ui/src/Components/EditProfile.jsx

```diff
@@ -29,6 +38,7 @@ class CreateProfile extends React.Component {
             govId: null,
             toolPicFlag: false,
             toolPic: null,
+            readyForVerification:0,
```

Comment:
Technically, does readyForVerification:0 refer to both "Application began" and "Verified"?

Reply:
WDYM? Based on boolean rules 0 = false, 1 = true. 
Is it it ready for verification? 
No, then the application is still pending then "Application Began". 
Yes, then the application is ready to be submitted then "Application Pending Employer Validation"

Reply:
There will never be a state in this view that will toggle to Verified - admins cannot edit profiles

Reply:
Ah rereading this, I get what you mean. In this component, Verified will never be assigned. Since this a client only view. Admin will not have access to this view so it will only refer to "Application Began" (if 0) and "Application Pending Verification" (if 1)

Reply:
makes sense!